pulp-glue-ostree/pulp_glue/ostree/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0.dev"
+__version__ = "0.4.0.dev"
```

Comment:
It's missing from https://github.com/pulp/pulp-cli-ostree/blob/main/.bumpversion.cfg. So yes you need to sync it manually at least now.

But at this point rather than fixing .bumpversion.cfg, I'd suggest running
`../pulp-cli/cookiecutter/apply_templates.py` to upgrade to using bump-my-version.